From b71fae852b2b3ecf45b3428d78ceac3ea8393f53 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 25 Jul 2018 18:48:58 +0000
Subject: [PATCH] TRITON-508 config-agent autometadata should contain an array
 of IPs for each nic (bump version) Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 5a00fc0..701079c 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "config-agent",
     "description": "SmartDataCenter Config Agent",
-    "version": "1.7.1",
+    "version": "1.8.0",
     "author": "Joyent (joyent.com)",
     "dependencies": {
         "assert-plus": "0.1.5",
-- 
2.21.0

